File: articles/automation-bias-and-human-overreliance/prolog/automation_bias_risk.pl
automation_bias_risk(Acceptance, Quality, Uncertainty, ReviewDeficit, OverrideFriction, WeakContestability, Score) :-
    Gap0 is Acceptance - Quality,
    OverrelianceGap is max(0, Gap0),
    Score is (Acceptance + OverrelianceGap + Uncertainty + ReviewDeficit + OverrideFriction + WeakContestability) / 6.

% Query example:
% ?- automation_bias_risk(0.93, 0.71, 0.29, 0.65, 0.72, 0.0, Score).

File: articles/automation-bias-and-human-overreliance/julia/automation_bias_risk.jl
acceptance = 0.93
quality = 0.71
uncertainty = 0.29
review_deficit = 0.65
override_friction = 0.72
weak_contestability = 0.0
score = (acceptance + max(0, acceptance - quality) + uncertainty + review_deficit + override_friction + weak_contestability) / 6
println("automation_bias_risk_score=", round(score, digits=4))
